Assets/GothicVR/Scripts/Debugging/FeatureFlags.cs

```diff
@@ -33,6 +33,8 @@ public enum SunMovementPerformance
 
         [Header("__________DayTime__________")]
         public bool enableDayTime;
+        
+        [Range(0.5f, 300f)] public float TimeMultiplier;
```

Comment:
What is it for?
Is it playing ingame time much faster?
YES? - Then please add a comment
NO? - Please rename like "cloud speed multiplier" or whatever it is doing

Reply:
YES
i'm adding a tooltip to better clarify this